tool/tsh/tsh.go

```diff
@@ -341,6 +341,8 @@ type CLIConf struct {
 	// AWSEndpointURLMode is an AWS proxy mode that serves an AWS endpoint URL
 	// proxy instead of an HTTPS proxy.
 	AWSEndpointURLMode bool
+	// KeyStore is the path to a Java KeyStore.
```

Comment:
Please call out the format of the keystore here too.

Reply:
As mentioned above, I would consider `KeyStore` is the format.

Reply:
I think maybe the confusion here is I only want to support the [Java KeyStore](https://docs.oracle.com/javase/7/docs/api/java/security/KeyStore.html), not any generic keystore?

Reply:
Same as the other comment, let's call out whether this is jks, jceks, both, etc.